catkin_ws/src/propulsion/src/thrust_mapper.py

```diff
@@ -19,27 +19,17 @@
 # D_1 = 0.895 #m
 # D_2 = 0.778 #m
 
-# T = np.matrix(
-#         [[1., 1., 0., 0., 0., 0., 0., 0.],
-#         [0., 0., 1., -1., 0., 0., 0., 0.],
-#         [0., 0., 0., 0., -1., -1., -1., -1.],
-#         [0., 0., 0., 0., -d/2, d/2, d/2, -d/2],
-#         [0., 0., 0., 0., -D_2/2, -D_2/2, D_2/2, D_2/2],
-#         [-d/2, d/2, D_1/2, D_1/2, 0., 0., 0., 0.]]
-#         )
-
-
 d = 0.284 #m
 hypot = math.sqrt(math.pow(0.672/2,2) + math.pow(0.258/2,2)) #m
 D_2 = 0.571 #m
 
 T = np.matrix(
-        [[-1., -1., 0., 0., 0., 0., 0., 0.],
-        [0., 0., 1., -1., 0., 0., 0., 0.],
-        [0., 0., 0., 0., -1., -1., -1., -1.],
-        [0., 0., 0., 0., -d/2, d/2, d/2, -d/2],
-        [0., 0., 0., 0., -D_2/2, -D_2/2, D_2/2, D_2/2],
-        [d/2, -d/2, hypot, hypot, 0., 0., 0., 0.]]
+        [[-1.,  -1.,  0.,    0.,    0.,     0.,     0.,     0.],
```

Comment:
Make sure this is updated with recent changes to the thrust mapper matrix, when I merge it will revert those changes if they are on noetic